gameserverstatus/gameserverstatus.py

```diff
@@ -233,20 +233,23 @@ async def do_status_ss14(self, ctx: Messageable, cfgname: str, dat: Dict[str, st
 
         async with aiohttp.ClientSession() as session:
             async with session.get(addr + "/status") as resp:
-                json =await resp.json()
+                json = await resp.json()
 
             count = json["players"]
+            countmax = json["soft_max_players"]
             name = json["name"]
+            round_id = json["round_id"]
+            gamemap = json["map"]
```

Comment:
Maybe make sure these are `.get()` so that if the server doesn't report them it's not an error.